go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/lightninglabs/gozmq v0.0.0-20191113021534-d20a764486bf
 	github.com/lightninglabs/neutrino v0.16.0
 	github.com/lightninglabs/neutrino/cache v1.1.2
+	github.com/lightningnetwork/lnd/fn v1.2.5
```

Comment:
didn't realize we were importing modules from `lnd` before😮‍💨

Reply:
Yeah. But those are all submodules, so it shouldn't really matter. But noticed this actually needs to be `github.com/lightningnetwork/lnd/fn/v2`, so I changed that.